api4/user.go

```diff
@@ -44,6 +44,7 @@ func (api *API) InitUser() {
 	api.BaseRoutes.User.Handle("/password", api.ApiSessionRequired(updatePassword)).Methods("PUT")
 	api.BaseRoutes.User.Handle("/promote", api.ApiSessionRequired(promoteGuestToUser)).Methods("POST")
 	api.BaseRoutes.User.Handle("/demote", api.ApiSessionRequired(demoteUserToGuest)).Methods("POST")
+	api.BaseRoutes.User.Handle("/verify", api.ApiSessionRequired(verifyUser)).Methods("PUT")
```

Comment:
Being this an action and not an update, we should use `POST` to follow REST:

```suggestion
	api.BaseRoutes.User.Handle("/verify", api.ApiSessionRequired(verifyUser)).Methods("POST")
```